src/components/ItemDetail/ItemDetail.tsx

```diff
@@ -81,6 +78,11 @@ function Revision({ id, release, ring, body }: RevisionProps) {
             <Edit />
           </a>
         )}
+        {teams && (
```

Comment:
If `teams` is an empty array, so the value is `[]`, the condition will be true and the `<Teams>` Component is rendered. I guess, this component should only be rendered if there are some values in the array, right? Therefor, I suggest to validate that the array has a length greater than 0.

```
// The !! is not necessarily required, because the type of the variable is already captured via TS
{!!teams && teams.length > 0  && <div....}
```